lib/discourse_chat/provider/gitter/gitter_provider.rb

```diff
@@ -0,0 +1,30 @@
+module DiscourseChat
+  module Provider
+    module GitterProvider
+      PROVIDER_NAME = 'gitter'.freeze
+      PROVIDER_ENABLED_SETTING = :chat_integration_gitter_enabled
+      CHANNEL_PARAMETERS = [
+        { key: "name", regex: '^\S+$', unique: true },
+        { key: "webhook_url", regex: '^https://webhooks.gitter.im/e/\S+$', unique: true, hidden: true }
```

Comment:
Very tiny thing here - a `.` in a regular expression allows any character. So technically

```
https://webhooksAgitterAim/e/blah
```
would match.

Solution is just to escape both dots with a backslash